### DISCUSSION TITLE
Variants of the "withdraw" method

### DISCUSSION THREAD

**Discussion by razgraf** [sablier-labs/evm-monorepo#6] [General]
I'm proposing we add a variant to the withdraw method that allows routing claimed funds to a specified address.

```solidity
function withdrawTo(uint256 streamId, uint256 amount, address destination) external;
```

DX: It might be useful to have this for certain proxy patterns, erc4626 or purely integration utility
UX: It allows us to add an extra (advanced) field to the withdraw form where a user could specify the destination of funds

**Comment by PaulRBerg** [comment]
All of these are available now:

- `withdraw`
- `withdrawTo`
- `withdrawAll`
- `withdrawAllTo`

So I'm locking this conversation.
